src/Surfnet/Stepup/Tests/Configuration/Event/AppointedAsRaaForInstitutionEventTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Copyright 2016 SURFnet B.V.
```

Comment:
Nitpick: user current year (also goes for other new files added in this PR)

Reply:
changed